src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -18,14 +18,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System.Runtime.Caching" />
-    <Reference Include="VSLangProj140">
-      <HintPath>$(NuGetPackageRoot)vslangproj140\14.0.25029\lib\net20\VSLangProj140.dll</HintPath>
```

Comment:
If this is no longer necessary, it should probably get removed in bootstrap.proj.

Reply:
Do you mean packages.targets?

Reply:
Oh yeah, cause it was removed from bootstrap.proj a while back.

Reply:
Removed from packages.targets.